manila/share/manager.py

```diff
@@ -4606,6 +4606,10 @@ def _wrapped_delete_share_server():
 
             self.db.share_server_update(context, server_id,
                                         {'status': constants.STATUS_DELETING})
+
+            LOG.debug("Deleting share server network'%s'", share_server['id'])
```

Comment:
nit: probably it is easier to understand that we log the id of the share server and not something of the network, anyway a whitespace was missing

```suggestion
            LOG.debug("Deleting network of share server '%s'", share_server['id'])
```